Description: Don't break compiler security flags.
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 tv-grab-dvb (0.9-0cyber1) experimental; urgency=low
 .
    * Initial release.
Author: Trent W. Buck <trentbuck@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: tv-grab-dvb_0.9/Makefile
===================================================================
--- tv-grab-dvb_0.9.orig/Makefile	2014-05-13 18:55:32.000000000 +1000
+++ tv-grab-dvb_0.9/Makefile	2014-05-13 19:00:15.840607556 +1000
@@ -1,15 +1,7 @@
-#Perhaps you want a line like this instead. I've not used autoconf yet
-#CFLAGS=-Wall -O2 -I/usr/src/dvb-kernel/linux/include/
-CFLAGS=-Wall -O2
-
-tv_grab_dvb:	tv_grab_dvb.o crc32.o lookup.o
-	gcc -o tv_grab_dvb tv_grab_dvb.o crc32.o lookup.o
+CFLAGS += -Wall -O2
 
+tv_grab_dvb:    tv_grab_dvb.o crc32.o lookup.o
 tv_grab_dvb.o:  dvb_info_tables.h si_tables.h
-
-lookup.o:	lookup.h
-
+lookup.o:       lookup.h
 clean:
-	rm *.o tv_grab_dvb
-
-	
+	rm -f *.o tv_grab_dvb
